Resources/Prototypes/Entities/Objects/Fun/crayons.yml

```
@@ -21,7 +21,7 @@
       enum.CrayonUiKey.Key:
         type: CrayonBoundUserInterface
   - type: Crayon
-    capacity: 15
+    capacity: 45
```

```suggestion
    capacity: 25
```
Ok, turns out that decals can be quite expensive performance-wise. We can increase it a little bit to 25, but 45 is too high and would encourage people to spam them over the entire station. If you need more for making art you can still order another box at cargo when needed.